paradedb/benchmark.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# Cleanup function to reset the environment
+cleanup() {
+    echo ""
+    echo "Cleaning up..."
+    if sudo docker ps -q --filter "name=paradedb" | grep -q .; then
+        sudo docker kill paradedb
+    fi
+    sudo docker rm paradedb
+  echo "Done, goodbye!"
```

Comment:
```suggestion
    echo "Done, goodbye!"
```

Reply:
I pushed another commit to shorten the description, and also indented properly this block. Thank you for flagging!